### ISSUE TITLE
Q&A section not displaying

### ISSUE THREAD

**Issue by GaTechThomas** [MicrosoftDocs/vsts-docs#4527]
I've been trying to understand how build drafts works for Azure DevOps Services (not Server) but have been unable to find it. I finally searched the github repo for this documentation and discovered that it should be showing for the Azure DevOps filter for this page, but for some reason it is not. Q&A is completely gone when Azure DevOps Services is selected.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7336a04e-5b38-d02f-8bba-a55445b2f094
* Version Independent ID: 783c30b3-63b6-da7c-1336-b000bb8b4c66
* Content: [Create your first pipeline - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline?view=azure-devops-2019&tabs=tfs-2018-2)
* Content Source: [docs/pipelines/create-first-pipeline.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/create-first-pipeline.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @andyjlewis
* Microsoft Alias: **alewis**

**Comment by WilliamAntonRohm** [comment]
@GaTechThomas -- thank you for bringing this to our attention.

@andyjlewis -- please check to see if the Q&A section should be appearing in that article for all versions of the topic.

**Comment by GaTechThomas** [comment]
Thank you. It also appears that many sections from Azure Devops Server filter should appear in the Azure DevOps Services filter results.

**Comment by steved0x** [comment]
@GaTechThomas Thank you for letting us know and sorry for the delay. I am glad that you were able to find the content you needed even if it wasn't ideal. I have created an internal work item to restore the Q/A section for this article when Azure DevOps Services is selected in the version switcher, once I can review it to ensure that it is up to date for the current version. It looks like it is, with possibly some minor tweaks to wording. I have linked that work item to this GH issue, so when the changes are rolled it it will be linked to this issue.
